src/frontend/screens/ObservationEdit/SaveButton.tsx

```diff
@@ -69,20 +71,43 @@ export const SaveButton = ({
   const navigation = useNavigationFromRoot();
   const createObservationMutation = useCreateObservation();
   const editObservationMutation = useEditObservation();
+  const createAttachmentsMutation = useCreateAttachmentsMutation();
```

Comment:
on line 171 we need add 'isPending` state to the button.

Reply:
addressed via https://github.com/digidem/CoMapeo-mobile/pull/187/commits/a499581e2fa5d709603999dae356f1410a0a456e